packages/app/src/components/nominationInfo/applicationDetail.js

```diff
@@ -1,22 +1,22 @@
-import React from "react";
-import styles from "./styles.module.css";
+import React from 'react';
+import styles from './styles.module.css';
 
 function ApplicationDetail(props) {
-      return (
-        <div className={styles.main}>
-          <div className={styles.header}>
-          <label className={styles.bold}>{props.title}</label>
+  return (
+    <div className={styles.main}>
+      <div className={styles.header}>
+        <label className={styles.title}>{props.title}</label>
+      </div>
+      <div className={[styles.content, props.gridContent && styles['grid-container']].join(' ')}>
+        {props.fields.map((obj) => (
+          <div key={obj.label} className={obj.label === '' ? styles.mobileHide : ''}>
+            <label className={styles.label}>{obj.label}</label>
+            <span className={styles.value}>{String(obj.value)}</span>
```

Comment:
Shouldn't need to wrap this in a `String()` call

Reply:
@bwalborn looks like the String() call is still here. Were you able to figure out what `obj.value` is and how to properly display it as a string?

Reply:
The value should be a string. Not sure how else to I should display the value here. Maybe you can help me out with this?